openbb_terminal/sdk.py

```diff
@@ -20,6 +20,8 @@
     controllers as ctrl,
     models as model,
 )
+from openbb_terminal import feature_flags as obbff
```

Comment:
Why are these items being added? Line 2 of the file says explicitly that this file is auto generated.

Reply:
While Running this Command "python generate_sdk.py sort"
These lines are automatically generated.